iset.mm

```diff
@@ -64043,6 +64073,177 @@ would need to be strengthened (for example, to imply included middle) to
+        AWRUOGXBJUPWMXCWSVSWLUKUQURUSUTWBWOCQWIWBCWOWBCWOQZWMCOZWAXELWMVTOWAXEV
+        AVSWLVBWMVTCVCVDVELXDXEVFWALXEXEKMWMVGZVHZKCVIZPXDLXHXELXGKCXFVJVKVLKCW
+        MVMVNVOVPVQVOVR $.
+
+      $( Lemma for ~ isbth .  (Contributed by NM, 27-Mar-1998.) $)
+      sbthlem7 $p |- ( ( Fun f /\ Fun `' g ) -> Fun H ) $=
+        ( cv wfun wa cres funres cdm cin c0 wss ax-mp ccnv cuni cdif wceq dmres
+        cun inss1 eqsstri ssrin sslin sstri disjdif sseqtri funun syl2an funeqi
+        ss0 mpan2 sylibr ) EKZLZFKUAZLZMUTDUBZNZVBBVDUCZNZUFZLZGLVAVELZVGLZVIVC
+        VDUTOVFVBOVJVKMVEPZVGPZQZRUDZVIVNRSVOVNVDVFQZRVNVDVMQZVPVLVDSVNVQSVLVDU
+        TPZQVDUTVDUEVDVRUGUHVLVDVMUITVMVFSVQVPSVMVFVBPZQVFVBVFUEVFVSUGUHVMVFVDU
+        JTUKVDBULUMVNUQTVEVGUNURUOGVHJUPUS $.
+
+      $( Lemma for ~ isbth .  (Contributed by NM, 27-Mar-1998.) $)
+      sbthlemi8 $p |- ( ( ( EXMID /\ Fun `' f )
+          /\ ( ( ( Fun g /\ dom g = B ) /\ ran g C_ A ) /\ Fun `' g ) )
+          -> Fun `' H ) $=
+        ( wem cv ccnv wfun wa cdm wceq crn cres cdif wss cuni funres11 ad2antlr
+        cun cin c0 funcnvcnv syl ad2antrr ad2antrl simpll simprll simprd simprr
+        simprlr cima df-ima df-rn eqtr2i eqtri sbthlemi4 syl5eqr ineq12 sylancr
+        w3a disjdif syl6eq syl121anc funun syl21anc cnveqi cnvun funeqi sylibr
+        ) KELZMNZOZFLZNZVSPCQZOZVSRBUAZOZVSMZNZOZOZVPDUBZSZMZWEBWITZSZMZUEZNZGM
+        ZNWHWKNZWNNZWKPZWNPZUFZUGQZWPVQWRKWGWIVPUCUDWDWSVRWFVTWSWAWCVTWEMNWSVSU
+        HWLWEUCUIUJUKWHKWAWCWFXCKVQWGULWHVTWAVRWBWCWFUMUNVRWBWCWFUPVRWDWFUOKWAW
+        COWFVFZXBVPWIUQZCXETZUFZUGXDWTXEQXAXFQXBXGQXEWJRWTVPWIURWJUSUTXDXAWEWLU
+        QZXFXHWMRXAWEWLURWMUSVAABCDEFHIVBVCWTXEXAXFVDVEXECVGVHVIWKWNVJVKWQWOWQW
+        JWMUEZMWOGXIJVLWJWMVMVAVNVO $.
+
+      $( Lemma for ~ isbth .  (Contributed by NM, 28-Mar-1998.) $)
+      sbthlemi9 $p |- ( ( EXMID /\ f : A -1-1-> B /\ g : B -1-1-> A ) ->
+                    H : A -1-1-onto-> B ) $=
+        ( wem wfn wfun cdm wceq wa sylib simpld df-fn simprd cv wf1 w3a crn wss
+        ccnv wf1o wf simp2 df-f1 df-f simp3 sbthlem7 syl2anc sbthlemi5 syl12anc
+        simp1 sylanbrc jca sbthlemi8 syl22anc df-rn sbthlemi6 syl5eqr dff1o4 )
+        KBCEUAZUBZCBFUAZUBZUCZGBLZGUFZCLZBCGUGVJGMZGNBOZVKVJVFMZVHUFMZVNVJVPVFN
+        BOZVJVFBLZVPVRPVJVSVFUDCUEZVJBCVFUHZVSVTPVJWAVFUFMZVJVGWAWBPKVGVIUIBCVF
+        UJQZRBCVFUKQZRVFBSQZRVJCBVHUHZVQVJVIWFVQPKVGVIULCBVHUJQZTZABCDEFGHIJUMU
+        NVJKVRVHUDBUEZVOKVGVIUQZVJVPVRWETVJVHCLZWIVJWFWKWIPVJWFVQWGRCBVHUKQZTZA
+        BCDEFGHIJUOUPGBSURVJVLMZVLNZCOZVMVJKWBVHMZVHNCOZPZWIPVQWNWJVJWAWBWCTVJW
+        SWIVJWKWSVJWKWIWLRVHCSQZWMUSWHABCDEFGHIJUTVAVJKVTWRWIPZVQWPWJVJVSVTWDTV
+        JWRWIVJWQWRWTTWMUSWHKVTPXAVQPPWOGUDCGVBABCDEFGHIJVCVDVAVLCSURBCGVEUR $.
+
+      ${
+        $d f g A $.  $d f g B $.
+        sbthlem.4 $e |- B e. _V $.
+        $( Lemma for ~ isbth .  (Contributed by NM, 28-Mar-1998.) $)
+        sbthlemi10 $p |- ( ( EXMID /\ ( A ~<_ B /\ B ~<_ A ) ) -> A ~~ B ) $=
```

Comment:
Why are some lemmas called sbthlem* and some sbthlemi* ? What is the meaning of "i"? Can it be omitted?

Reply:
Ah, I see: "i" means "intuitionistic version of" (additionally requiring EXMID as antecedent compared with the non-intuitionistic version). Maybe this should be explained in the first concerned lemma.

Reply:
The rule is "Theorems which are the same as in set.mm should be named the same (that is,
where the statement of the theorem is the same; the proof can differ without
a new name being called for).  Theorems which are different should be named
differently" (quote from a comment near the top of iset.mm). As for the `i` this convention is used here and there starting with https://us.metamath.org/ileuni/ax-i9.html or so although I try to name things another way if I can think of a better way in a particular situation.

Reply:
Can you add the usage of an "i" to provide a different name as described in "1. Naming" in that section? For example:

> Theorems which are different should be named differently (although if additional hypotheses are added in iset.mm
> the name need not be changed). **Often, an "i" is added to achieve this (see, for example, ~sbthlemi10 versus sbthlem10 in set.mm).**

Reply:
I'll make this a separate pull request.

Reply:
Submitted as https://github.com/metamath/set.mm/pull/2769